pytest_pootle/fixtures/views.py

```diff
@@ -294,6 +318,29 @@ def tp_views(request, client, request_users):
     return test_type, tp, response.wsgi_request, response, kwargs
 
 
+@pytest.fixture(params=("browse", "translate", "export"))
+def view_types(request):
+    """List of possible view types."""
+    return request.param
+
+
+@pytest.fixture(params=DISABLED_PROJECT_URL_PARAMS.keys())
+def dp_view_urls(request, view_types):
+    """List of url params required for disabled project tests."""
+    from pootle_project.models import Project
+
+    disabled_project = Project.objects.get(code="disabled_project0")
+    dp_url_params = DISABLED_PROJECT_URL_PARAMS[request.param].copy()
+    view_name = dp_url_params.pop("view_name")
+    kwargs = {
+        "project_code": disabled_project.code,
```

Comment:
Also ideally the `project_code` is better placed in `DISABLED_PROJECT_URL_PARAMS`, even if it is repeated a lot of times.